src/config.ts

```diff
@@ -50,6 +57,11 @@ export function getCommonConfig(): CommonConfig {
       region,
       maxRetries: maxRetries,
     },
+    identity: {
+      stack: getRequiredEnv("STACK"),
```

Comment:
I don't think `STACK` and `APP` are currently being set as environment variables? I guess `STACK` can be added [here](https://github.com/guardian/cloudwatch-logs-management/blob/main/template.yaml#L72-L77) and `APP` should be added for individual functions?

Reply:
Great catch! I was looking at [tags](https://github.com/guardian/cloudwatch-logs-management/blob/d74dde107601ce0496b4f2c7f0943a981d4226af/template.yaml#L126-L128)! I'll update the template too!

Reply:
Looking at an instance of a lambda in our account, the tags are already present 🤔 

![image](https://user-images.githubusercontent.com/836140/139837081-cbefdd71-0b7c-435d-930f-d760ea94f2a2.png)

Reply:
Oops, again these are tags not env vars! Ignore me!